docs/developers/HowTo.md

```diff
@@ -54,7 +54,7 @@ ${GLUTEN_HOME}/dev/builddeps-veloxbe.sh --build_tests=ON --build_benchmarks=ON -
 
 ```
 cd ${GLUTEN_HOME}
-mvn clean package -Pspark-3.2 -Pbackends-velox -Pceleborn -Puniffle
+mvn clean package -Pspark-3.2 -Pbackends-velox -Pceleborn -Puniffle -DskipTests
```

Comment:
We can remove the above command, and just keep the below mvn test command.

Reply:
Done